packages/storybook/stories/components/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import { ArgsTable, Description, Primary, PRIMARY_STORY, Stories } from '@storybook/addon-docs';
+import { Meta, StoryObj } from '@storybook/react';
+import { ButtonGroup } from '@utrecht/component-library-react/dist/css-module';
+import readme from '@utrecht/components/badge-counter/README.md?raw';
+import tokensDefinition from '@utrecht/components/badge-counter/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import clsx from 'clsx';
+import parse from 'html-react-parser';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const meta = {
+  title: 'CSS Component/Button Group',
+  id: 'css-button-group',
+  component: ButtonGroup,
+  args: {
+    innerHTML: `<button class='utrecht-button utrecht-button--primary-action'>Save and continue</button><button class='utrecht-button utrecht-button--secondary-action'>Back</button>`,
```

Comment:
I suspect the button group stories were implemented in a time before we had reusable code snippets in `story-template.jsx`, and that's why there is this `innerHTML` code.

If you don't see any obvious objections, I suggest we simply use `<Button>` component here.